server/src/main/java/org/opensearch/ratelimitting/admissioncontrol/AdmissionControlService.java

```diff
@@ -101,4 +123,19 @@ public List<AdmissionController> getAdmissionControllers() {
     public AdmissionController getAdmissionController(String controllerName) {
         return this.ADMISSION_CONTROLLERS.getOrDefault(controllerName, null);
     }
+
+    /**
+     * Return admission control stats
+     */
+    public AdmissionControlStats stats() {
+        List<AdmissionControllerStats> statsList = new ArrayList<>();
+        if (this.ADMISSION_CONTROLLERS.size() > 0) {
+            this.ADMISSION_CONTROLLERS.forEach((controllerName, admissionController) -> {
+                AdmissionControllerStats admissionControllerStats = new AdmissionControllerStats(admissionController, controllerName);
```

Comment:
Ideally AdmissionController should have member `name` so we dont need to always pass `name` along with it. Its fine to maintain duplicate name as key in hashmap for quick reference lookup.

Reply:
Yes admissionController have the name. we don't need to explicitly pass it. will update it.